curriculum/get-challenges.js

```diff
@@ -289,6 +289,11 @@ function generateChallengeCreator(lang, englishPath, i18nPath) {
     challenge.blockType = meta.blockType;
     challenge.hasEditableBoundaries = !!meta.hasEditableBoundaries;
     challenge.order = meta.order;
+
+    if (!challenge.description) challenge.description = '';
+    if (!challenge.instructions) challenge.instructions = '';
+    if (!challenge.seed) challenge.seed = '';
```

Comment:
```suggestion
```
Oops. The seed is already checked for (as part of the challenge file validation).